ldelf/ldelf.ld.S

```diff
@@ -27,6 +27,7 @@ SECTIONS {
 		*(.vfp11_veneer)
 		__text_end = .;
 	}
+	.note.gnu.property : { *(.note.gnu.property) }
```

Comment:
Since the core doesn't check the property, I think it would be better to simply add this to `/DISCARD/`.

Reply:
Sure, I will test that and update the PR.

@jforissier , I tried compiling my QEMUv8 setup with    COMPILE_S_USER=32 in the command line. The build stops at optee-examples
Building /home/ruchika/OPTEE-EXTENSIONS/out-br/build/optee_examples_ext-1.0/acipher/ta/Makefile
make[3]: *** No rule to make target 'all'.  Stop.
make[2]: *** [package/pkg-generic.mk:271: /home/ruchika/OPTEE-EXTENSIONS/out-br/build/optee_examples_ext-1.0/.stamp_built] Error 2
make[1]: *** [Makefile:23: _all] Error 2
make[1]: Leaving directory '/home/ruchika/OPTEE-EXTENSIONS/out-br'
make: *** [common.mk:325: buildroot] Error 2

Any ideas what could be going wrong  here ?

Reply:
No idea, `make COMPILE_S_USER=32 run` should work. Try `rm -rf ../out-br/build/optee_* ../optee_os/out` first in case some leftovers from the 64-bit build would be problematic.

Reply:
Sorry for the noise. I had another option on in the build line that was causing the issues. Resolved now,

Reply:
Ok..will do that in both core as well as ldelf lds